utils/docker/Dockerfile.jammy

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:jammy
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TZ=America/Los_Angeles
+ARG DOCKER_IMAGE_NAME_TEMPLATE="mcr.microsoft.com/playwright/java:v%version%-jammy"
+
+# === INSTALL JDK and Maven ===
+
+RUN apt-get update && \
+    # Install install jdk 17 in a separate apt-get command so that
+    # installing maven doesn't bring in jdk 11
+    apt-get install -y --no-install-recommends openjdk-17-jdk && \
+    apt-get install -y --no-install-recommends \
+    # Ubuntu 22.04 and earlier come with Maven 3.6.3 which fails with
+    # Java 17, so we install latest Maven from Apache instead.
+    # maven \
+    # Install utilities required for downloading browsers
+    curl \
+    # Install utilities required for downloading driver
+    unzip \
+    # For the MSEdge install script
+    gpg && \
+    rm -rf /var/lib/apt/lists/* && \
+    # Create the pwuser
+    adduser pwuser
+
+RUN VERSION=3.8.8 && \
```

Comment:
jammy may have fresh enough Maven and we may be able to drop this lines

Reply:
Will keep them in sync for now!